### ISSUE TITLE
Cannot build the plugin

### ISSUE THREAD

**Issue by Peter2121** [oz123/coredns-netbox-plugin#4]
Building in FreeBSD 12.2 jail with go 1.15.6 as plugin of CoreDNS 1.8.0:

```
...
# github.com/oz123/coredns-netbox-plugin
../coredns-netbox-plugin/setup.go:28:31: cannot use setup (type func(*"github.com/caddyserver/caddy".Controller) error) as type "github.com/coredns/caddy".SetupFunc
in argument to plugin.Register
../coredns-netbox-plugin/setup.go:43:20: undefined: metrics.MustRegister
../coredns-netbox-plugin/setup.go:48:21: cannot use c (type *"github.com/caddyserver/caddy".Controller) as type *"github.com/coredns/caddy".Controller in argument to
 dnsserver.GetConfig
*** Error code 2

```

**Comment by Peter2121** [comment]
It seems that two errors can be fixed changing import from `github.com/caddyserver/caddy` to `github.com/coredns/caddy`
[https://github.com/Denemor/k8s_gateway/commit/a4b8fe7b2e237d80cf8c605fee26782b43daa5b3](https://github.com/Denemor/k8s_gateway/commit/a4b8fe7b2e237d80cf8c605fee26782b43daa5b3)

**Comment by oz123** [comment]
Thanks, I try and patch it.

**Comment by oz123** [comment]
Fixed in 3bbb40956cee371ea83eea701a4669a095248a22.
